### PR TITLE
Changes to 'Check if you need a visa' 

### PR DESCRIPTION
**Pull request by deborahchua**
There are some changes required on 31 December.

The main changes are:

+ remove additional question for EEA citizens about pre 1 Jan travel
+ remove questions relating to Article 10/20 cards
+ update questions and outcomes relating to settlement and British citizenship

Trello card: https://trello.com/c/RiQrUIbO/2256-5-changes-to-check-if-you-need-a-uk-visa

Paired with @beccapearce 

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: lib/smart_answer_flows/check-uk-visa.rb [hunk 1]
```diff
@@ -235,7 +235,7 @@ def define
 
         next_node do |response|
           if response == "yes"
-            question :article_10_card?
```
Comment (ChrisBAshton) [inline lib/smart_answer_flows/check-uk-visa.rb:258]: The [commit](https://github.com/alphagov/smart-answers/pull/5048/commits/1c613ca336a2b4d14c69d5e9ba6c90abb139d6a4) message says this question has been removed, but it doesn't happen until the next commit - should these two commits be squashed?

File: test/integration/smart_answer_flows/check_uk_visa_test.rb [hunk 2]
```diff
@@ -814,14 +814,13 @@ class CheckUkVisaTest < ActiveSupport::TestCase
...
         end
 
-        context "travelling with a a UK family member with an article 10 card" do
+        context "travelling with a a UK family member" do
           setup do
             add_response "yes"
-            add_response "yes"
           end
 
           should "take you to no visa needed outcome" do
```
Comment (ChrisBAshton) [inline test/integration/smart_answer_flows/check_uk_visa_test.rb:822]: Does this need rephrasing too?

File: test/integration/smart_answer_flows/check_uk_visa_test.rb [hunk 3]
```diff
@@ -782,7 +782,7 @@ class CheckUkVisaTest < ActiveSupport::TestCase
     context "coming to join family member, without an article 10 card, and they're a British citizen" do
       setup do
         add_response "family"
-        add_response "no"
+        # add_response "no"
```
Comment (ChrisBAshton) [inline test/integration/smart_answer_flows/check_uk_visa_test.rb:785]: Is this a mistake?

**Review by ChrisBAshton — APPROVED** [review]
LGTM 👍

**Comment by deborahchua** [comment]
> LGTM 👍

Thanks @ChrisBAshton ! I've made those changes and squashed the commits suggested if you have time to give it a re-review :)

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**